src/devices/bus.jl

```diff
@@ -0,0 +1,49 @@
+const pm_bustypes = Dict{PSY.BusTypes, Int}(
```

Comment:
move this to a definitions file and const definition uses capitalization. 

```suggestion
const PM_BUSTYPES = Dict{PSY.BusTypes, Int}(
```

we could do a match of our BusTypes in PSY for consistency.